documentation/modules/ROOT/pages/index.adoc

```diff
@@ -27,39 +22,38 @@ xref:samm-specification:ROOT:index.adoc[Specification]
 [.icon-cli]
 --
 [.title]
-SAMM Command Line Interface (SAMM-CLI)
+Aspect Model Editor
```

Comment:
Aspect Model Editor (AME)

Reply:
Would recommend to avoid introducing the AME acronym here. The acronym deliberately has been avoided so far throughout the whole documentation set. So there is no use in providing/explaining it here plus we would stick to the general good practice of avoiding acronyms where possible.